src/assets/config_files/usecases.json

```diff
@@ -47,6 +47,26 @@
           }
         ]
       },
+      {
+        "title": "Predict protein interaction surface with the ARDOCK server",
+        "description": "In silico analysis of your protein surface using molecular probes to reveal potential interaction sites",
+        "experience": ["all"],
+        "maturity": ["beta"],
+        "disabled": false,
+        "picture": {
+          "src": "https://raw.githubusercontent.com/antonelepfl/usecases/dev/src/assets/images/ardock.png"
+        },
+        "next": "ta_form",
+        "dataprotected": true,
+        "files": [
+          {
+            "entryname": "ArDock",
```

Comment:
This is going to be the 'nav item' that the user will get into the Collab. Do you think that name is representative enough? Could it be Detection Interaction? as you prefer.